x-pack/docs/build.gradle

```
@@ -56,6 +56,7 @@ testClusters.integTest {
   setting 'xpack.security.authc.realms.pki.pki1.certificate_authorities', '[ "testClient.crt" ]'
   setting 'xpack.security.authc.realms.pki.pki1.delegation.enabled', 'true'
   setting 'xpack.security.authc.realms.saml.saml1.order', '4'
+  setting 'xpack.security.authc.realms.saml.saml1.sp.logout', 'https://kibana.example.com/logout'
```

nit: usually `acs` and `logout` endpoints are on the same domain

```suggestion
  setting 'xpack.security.authc.realms.saml.saml1.sp.logout', 'https://kibana.org/logout'
```